src/utils/getImage.js

```diff
@@ -0,0 +1,29 @@
+const { default: axios } = require("axios");
+const cheerio = require("cheerio");
+const defaultImage = 'https://energy-price-news.nosweat.cloud/img/energy-prices-api-socials.png'
+
+async function getImage(url) {
+    if (/^(https:\/\/12ft.io\/)/.test(url)) {
+
+        url = url.slice(16);
+
+    } else if (url.indexOf('http') !== url.lastIndexOf('http') 
+        || url.match(/:\/\//) === null) {
+
+        console.log(`Tripped on ${url}`)
+        return defaultImage;
+
+    }
+    const response = await axios.get(url);
+    const $ = cheerio.load(response.data);
+
+    image = $('head').find('meta[property="og:image"]').attr('content');
```

Comment:
If you log this `image` variable, you'll see that it will be undefined.
Just add `<meta property="og:image" content="https://12ft.io/https://www.telegraph.co.uk/bills-and-utilities/renewable-energy/" />` in `__mocks__/axios.js`

Reply:
I have added it, but kept the `typeof image !== "undefined"` check on the getImage function because some of the articles that I was getting back on the real scrape did not have an og:image. Perhaps in the future I could extend to check for other types of meta tag image types like twitter etc.